code-for-canada/project-thundercat
# [Team Information] Populating team information tree view with api

# Description

This PR is introducing new API calls to get and populate the tree view in _Team information_ image description.

TODO(1): Populate the popup box description of the image description with markdown content (see PR #427)
TODO(2): Find a way to convert treeView variable to a local state

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Tree View (EN):**

![image](https://user-images.githubusercontent.com/23021242/60663561-93173780-9e2d-11e9-8137-c626e60a6d43.png)

**Tree View (FR):**

![image](https://user-images.githubusercontent.com/23021242/60663570-9a3e4580-9e2d-11e9-8fbc-ed585d3f14e1.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Click on Background tab.
3.  Click on _Your team_ on side nav.
4.  Open the image description.
5.  Make sure that the tree view looks exactly the same as before in both languages.

# Checklist
